docs/source/generate.rst

```diff
@@ -101,7 +100,7 @@ Browser
             loadFile("https://docxtemplater.com/tag-example.docx",function(error,content){
                 if (error) { throw error };
                 var zip = new PizZip(content);
-                var doc=new window.docxtemplater().loadZip(zip)
+                var doc=new window.docxtemplater(zip);
```

Comment:
Now that we use the new API, this line `new window.docxtemplater(zip)` can fail and throw, you should use the same catch handler as the one we use for doc.render(). 

Can you please refactor the catchhandler in one single function and call it for both try/catch ?